aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -972,6 +1024,8 @@ bool TensorIterator::fast_set_up() {
             //       We will revisit this logic when doing the TensorIterator refactor work and
             //       it would probably be better to move this logic out of TensorIterator.
 
+            // TODO(vitalyf): Potentionally might try to address bigger than original memory size
```

Comment:
making size sufficient is taken care of formerly in compute_shape and now in resize_outputs, so this is fine.